code/Model/SiteTree.php

```diff
@@ -2844,6 +2871,52 @@ public function isHomePage(): bool
         return $this->URLSegment === RootURLController::get_homepage_link();
     }
 
+    public function getDefaultSearchContext()
+    {
+        return SiteTreeSearchContext::create(
+            static::class,
+            $this->scaffoldSearchFields(),
+            $this->defaultSearchFilters()
+        );
+    }
+
+    public function searchableFields()
+    {
+        $fields = parent::searchableFields();
+        // Push `FilterClass` to the top of the fields list
+        $fields = array_merge([
+            'FilterClass' => [
+                'filter' => '',
+                'general' => false,
+                'title' => _t(__CLASS__ . '.Filter_Status', 'Page status'),
+                'field' => DropdownField::class,
+            ]
+        ], $fields);
+        return $fields;
+    }
+
+    public function scaffoldSearchFields($_params = null)
+    {
+        $fields = parent::scaffoldSearchFields($_params);
```

Comment:
```suggestion
    public function scaffoldSearchFields($params = null)
    {
        $fields = parent::scaffoldSearchFields($params);
```
Any reason for the `_` in the variable name?

Reply:
It matches the name in the parent class - since named arguments are a thing in PHP it's best to keep the names consistent across subclasses.